chain/cosmos/chain_node.go

```diff
@@ -863,7 +863,9 @@ func (tn *ChainNode) SubmitProposal(ctx context.Context, keyName string, prop Tx
 
 	command := []string{
 		"gov", "submit-proposal",
-		path.Join(tn.HomeDir(), file), "--gas", "auto",
+		path.Join(tn.HomeDir(), file),
+		"--gas", "auto",
+		"--chain-id", tn.Chain.Config().ChainID,
```

Comment:
Not necessary if we implement this chain-id flag in the TxCommand as stated here https://github.com/strangelove-ventures/interchaintest/pull/479#discussion_r1227345954